lca-cli/postpivot/postpivot.go

```diff
@@ -239,6 +246,63 @@ func (p *PostPivot) recert(ctx context.Context, seedReconfiguration *clusterconf
 	return nil
 }
 
+func (p *PostPivot) applyServerSSHKeys(serverSSHKeys []seedreconfig.ServerSSHKey) error {
+	if len(serverSSHKeys) == 0 {
+		p.log.Infof("No server ssh keys were provided, fresh keys already regenerated by recert, skipping")
+		return nil
+	}
+
+	if err := removeOldServerSSHKeys(); err != nil {
+		return fmt.Errorf("failed to remove old server SSH keys: %w", err)
+	}
+
+	for _, serverSSHKey := range serverSSHKeys {
+		if err := createServerSSHKeyFile(serverSSHKey); err != nil {
+			return fmt.Errorf("failed to create server SSH key file: %w", err)
+		}
+	}
+
+	return nil
+}
+
+func removeOldServerSSHKeys() error {
```

Comment:
maybe lets not bring them from seed?

Reply:
We can do that as part of https://issues.redhat.com//browse/MGMT-17988

Reply:
Will mention that in the ticket comments